src/static/profilo.html

```diff
@@ -84,7 +84,7 @@ <h2 id="mie-recensioni">Le mie recensioni</h2>
     </main>
 
     <footer>
-      <p>Copyright © 2025 Doflamingo. All rights reserved.</p>
+      <p lang="en">Copyright © 2025 Doflamingo. All rights reserved.</p>
```

Comment:
```suggestion
      <p>Copyright © 2025 Doflamingo. All rights reserved.</p>
```
Riporta il footer come prima visto che c'era la issue apposita. Sennò si creano conflitti